cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json

```diff
@@ -3,6 +3,7 @@
   "version": "<%= $CLI_PARAM_CustomPackageVersion %>",
   "description": "Cmf Custom <%= $CLI_PARAM_Tenant %> <%= $CLI_PARAM_CustomPackageName %> Package",
   "packageType": "Grafana",
+  "targetLayer": "grafana",
```

Comment:
is there any case where a user must specify this? this is usually hidden from the developer as the package type has a specific target layer, and we add it in the package type handler. e.g. https://github.com/criticalmanufacturing/cli/blob/f2af42440c9f050e360b85ce1bd76127e6be35b3/cmf-cli/Handlers/PackageType/BusinessPackageTypeHandler.cs#L27

seems to me this is missing in the package handler, not here.

Reply:
You are right.
It was an existing test case that led me in the wrong direction.
Check now please.